services/notifications.js

```diff
@@ -0,0 +1,21 @@
+export class Notifications {
+	constructor(settings) {
+		this.settings = settings;
+	}
+	push(data) {
+		const { text, type, expire } = data;
+		return new Promise ((resolve, reject) => {
+			this.settings.getSetNotificationsEnabled().then(val => {
```

Comment:
Nitpick.  `this.settings.getSetNotificationsEnabled()` returns a promise.  since you aren't using reject, you could have returned the promise returned by that function rather than creating a new chain.